radio/bluetooth/bluetooth.go

```diff
@@ -76,10 +79,66 @@ func Disconnect(client ble.Client) error {
 	return nil
 }
 
-func Scan(id string, timeout time.Duration) (map[string]bool, error) {
+func WriteChar(client ble.Client, char *ble.Characteristic, value []byte, noRsp bool) error {
```

Comment:
This function name is pretty misleading, in most languages char is short for character. Looking at only the function name (which forms the API) I'd think that this function is used to write a character to a bluetooth device stream.